apis/placement/v1alpha1/override_types.go

```diff
@@ -48,6 +48,21 @@ type ClusterResourceOverrideSpec struct {
 	// Policy defines how to override the selected resources on the target clusters.
 	// +required
 	Policy *OverridePolicy `json:"policy"`
+
+	// Placement defines whether the override is applied to a specific placement or not.
+	// If so, the override will trigger the placement rollout immediately when the rollout strategy type is RollingUpdate.
+	// Otherwise, it will be applied to the next rollout.
+	// +kubebuilder:validation:XValidation:rule="self == oldSelf",message="spec.placement is immutable"
+	// +optional
+	Placement *PlacementRef `json:"placement,omitempty"`
+}
+
+// PlacementRef is the reference to a placement.
+// For now, we only support ClusterResourcePlacement.
+type PlacementRef struct {
```

Comment:
we will have both CRP and RP, should this contain namespace too?

Reply:
There are four combinations,

RO attached to CRP
RO attached to RP, we won't support cross namespace at the beginning (security concern).
CRO attached to CRP
CRO attached to RP (not supported)

So that namespace won't be used. If needed, we can add later.